build/devops.yml

```diff
@@ -0,0 +1,101 @@
+name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+phases:
+
+- phase: windowsx86
+  displayName: 'Build Windows x86'
+  queue:
+    name: Hosted VS2017
+    demands:
```

Comment:
>demands [](start = 4, length = 7)

shouldn't the demands here list cmake too? Just curious

Reply:
Yeah, that's an interesting one. I generated this yaml from a build edited in DevOps.
And lo and behold cmake was in demands.
Then I scheduled the build and it complained that cmake requirement can not be met.
But the agents have it and the build runs if I remove the cmake demand.

So, can't do better than this ...

---
In reply to: [270525439](https://github.com/Azure/azure-macro-utils-c/pull/7#discussion_r270525439) [](ancestors = 270525439)